tests/protocol/test_wire.py

```diff
@@ -610,73 +611,6 @@ def test_get_ext_conf_with_uri(self, *args):
         self.assertEqual("BlockBlob", ext_conf.status_upload_blob_type)
         self.assertEqual(None, ext_conf.artifacts_profile_blob)
 
-    def test_save_or_update_goal_state_should_save_new_goal_state_file(self):
```

Comment:
these were tests for a trivial function that should have been private in the first place... no need for them